gapic/utils/case.py

```diff
@@ -45,3 +45,19 @@ def to_snake_case(s: str) -> str:
 
     # Done; return the camel-cased string.
     return s.lower()
+
+
+def to_camel_case(s: str) -> str:
+    '''Convert any string to camel case.
```

Comment:
nit: In this and the previous, pre-existing function, we're not touching spaces, right? Might be worth mentioning that.